game.rb

```diff
@@ -0,0 +1,91 @@
+class Game
+
+board={A1: " ",B1: " ",C1: " ",A2: " ",B2: " ",C2: " ",A3: " ",B3: " ",C3: " "}
+
+def self.print_board(board)
+  puts " | A | B | C |"
+  puts " -------------"
+  puts "1| #{board[:A1]} | #{board[:B1]} | #{board[:C1]} |"
+  puts " -------------"
+  puts "2| #{board[:A2]} | #{board[:B2]} | #{board[:C2]} |"
+  puts " -------------"
+  puts "3| #{board[:A3]} | #{board[:B3]} | #{board[:C3]} |"
+  puts " -------------"
+end
+
+def self.consistent_rows?(board)
```

Comment:
można to zrobić znacząco prościej, używając tzw. magicznego kwadratu.

http://www.cs.pomona.edu/~kim/CSC051S06/Lectures/Lecture23/Lecture23_1.html

http://mathforum.org/alejandre/magic.square/adler/adler.whatsquare.html

Pierwszy gracz który uzbiera 15 (tzn. suma pól na których są jego znaki) osiągnie 15 - będzie zwycięzcą.